### DISCUSSION TITLE
Ogg Vorbis: bitrate mode question

### DISCUSSION THREAD

**Discussion by TrixieDev** [libsndfile/libsndfile#913] [Q&A]
Looking at the ogg_vorbis.c source code, it seems that libsndfile always saves Ogg Vorbis files with a variable bit rate. Does that mean that the SFC_SET_BITRATE_MODE command is currently irrelevant for the Vorbis format?

I'd also like to know why, when I have saved a Vorbis file at VBR, the sf_current_byterate() function subsequently returns a value of 0, that is, SF_BITRATE_MODE_CONSTANT. Is this a bug, or is the implementation still lacking?

**Comment by evpobr** [comment]
Hi @TrixieDev . 

@SoapGentoo , can you help?

**Reply by SoapGentoo** [reply]
I have never looked at that part of the code, so I'm no expert
